pkg/model/manifest_test.go

```diff
@@ -0,0 +1,62 @@
+package model
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_GetBuildManifest(t *testing.T) {
+	tests := []struct {
+		name              string
+		manifestYAML      []byte
+		manifestNotExists bool
+		expectedErr       bool
+		expectedManifest  *ManifestBuild
+	}{
+		{
+			name: "exists-manifest",
+			manifestYAML: []byte(`build:
+  firstImage: ./first
```

Comment:
I would use a sample with a value for every field supported in the build section